packages/cicero-core/src/metadata.js

```diff
@@ -324,9 +384,22 @@ class Metadata {
     createTargetMetadata(runtimeName) {
         const packageJson = JSON.parse(JSON.stringify(this.packageJson));
         packageJson.accordproject.runtime = runtimeName;
-        return new Metadata(packageJson, this.readme, this.samples, this.request);
+        return new Metadata(packageJson, this.readme, this.samples, this.request, this.logo);
     }
 
+    /**
+     * Return the whole metadata content, for hashing
+     * @return {object} the content of the metadata object
+     */
+    toJSON() {
+        return {
+            'packageJson' : this.getPackageJson(),
+            'readme' : this.getREADME(),
+            'samples' : this.getSamples(),
+            'request' : this.getRequest(),
+            'logo' : JSON.stringify(this.getLogo()),
```

Comment:
I think I'd return the hash of the logo bytes here - or perhaps the logo bytes converted to a string. This is calling an implicit `toJSON` on the Buffer and I don't know how stable that is across Node versions.

Reply:
yes I'm unclear. I found some article:
https://medium.com/hackernoon/https-medium-com-amanhimself-converting-a-buffer-to-json-and-utf8-strings-in-nodejs-2150b1e3de57

Is doing the same by hand more or less reliable?

Reply:
This is documented. I'm not sure why this wouldn't work: https://nodejs.org/api/buffer.html#buffer_buf_tojson

Reply:
Note sure doing it by hand is better than relying on: https://github.com/nodejs/node/blob/cd1db2db48d28db805227e7f6ade79a106d85bc6/lib/buffer.js#L1073

Reply:
Still on this topic. Something like `buffer.toString('base64')` does seem like a good option too. @dselman is that what you had in mind?

Reply:
Finally settled on `buffer.toString('base64')` which should be stable and roundtrip. It also makes the result of `toJSON` easier to document or possibly more useful for applications.